docs/codeql/codeql-workshops/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,127 @@
+## Our Pledge
```

Comment:
You can remove this since it's in the repo root.

Reply:
done